### PR TITLE
Changing pfs info_endpoint again adding price and chain info

### PR DESCRIPTION
**Pull request by Dominik1999**

### CODE REVIEW

File: raiden/ui/app.py [hunk 1]
```diff
@@ -270,7 +270,9 @@ def run_app(
             click.secho(
                 f"'{pathfinding_service_info['message']}'. "
                 f"You have chosen pathfinding operator '{pathfinding_service_info['operator']}' "
-                f"with the running version '{pathfinding_service_info['version']}' ",
+                f"with the running version '{pathfinding_service_info['version']}' "
+                f"with chain_id:'{pathfinding_service_info['network_info']['chain_id']}."
```
Comment (palango) [inline raiden/ui/app.py]: I'd write "on chain_id"

File: raiden/ui/app.py [hunk 2]
```diff
@@ -270,7 +270,9 @@ def run_app(
             click.secho(
                 f"'{pathfinding_service_info['message']}'. "
                 f"You have chosen pathfinding operator '{pathfinding_service_info['operator']}' "
```
Comment (ulope) [inline raiden/ui/app.py]: This information should ideally also be logged. E.g. Could just be a simple `log.info('Using PFS', pfs_info=pathfinding_service_info)`

File: raiden/ui/app.py (hunk 1 above)
Comment (ulope) [reply raiden/ui/app.py]: There's a space missing after the period.

File: raiden/ui/app.py (hunk 2 above)
Comment (Dominik1999) [reply raiden/ui/app.py]: thanks done

File: raiden/ui/app.py (hunk 1 above)
Comment (Dominik1999) [reply raiden/ui/app.py]: thanks done

File: raiden/tests/unit/test_pfs_integration.py [hunk 3]
```diff
@@ -153,6 +158,10 @@ def test_get_pfs_info_success():
     with patch.object(requests, 'get', return_value=response):
         pathfinding_service_info = get_pfs_info("url")
 
+        assert pathfinding_service_info['price_info'] == 0
+        assert pathfinding_service_info['network_info']['chain_id'] == 1
+        assert pathfinding_service_info['network_info']['registry_address'] == \
```
Comment (LefterisJP) [inline raiden/tests/unit/test_pfs_integration.py]: style: We don't split lines with `\` in the codebase. IF needed to split it then do:

either of:

```python
registry_address = pathfinding_service_info['network_info']['registry_address']
assert registry_address ==  '0xB9633dd9a9a71F22C933bF121d7a22008f66B908'
```

```python
assert pathfinding_service_info['network_info']['registry_address'] == (
    '0xB9633dd9a9a71F22C933bF121d7a22008f66B908'
)
```

```

Comment (Dominik1999) [reply raiden/tests/unit/test_pfs_integration.py]: thanks, done

**Pull request merged**